### PR TITLE
[Camden] Add NSGRef to reports in JS and on server

### PR DESCRIPTION
**Pull request by chrismytton**
The JS asset changed are in the config file under the `asset_layers` key.

Part of https://github.com/mysociety/societyworks/issues/3305

### CODE REVIEW

**Comment by chrismytton** [comment]
It has been pointed out to me that this should be using the ✨ New Asset Configuration ✨  rather than being in JS, will change.

File: perllib/FixMyStreet/Cobrand/Camden.pm [hunk 1]
```diff
@@ -34,4 +34,42 @@ sub privacy_policy_url {
...
+    # uncoverable subroutine
+    # uncoverable statement
+    {
+        buffer => 1000, # metres
+        url => "https://tilma.mysociety.org/mapserver/camden",
+        srsname => "urn:ogc:def:crs:EPSG::27700",
+        typename => "Streets",
+        property => $property,
+        accept_feature => sub {
+            # Sometimes the nearest feature has a NULL streetref1 property
```
Comment (davea) [inline perllib/FixMyStreet/Cobrand/Camden.pm:49]: These `streetref1` references should be `NSG_REF`, based on L67.

**Review by davea — APPROVED** [review]
One tiny thing in the comment, but 👍

File: perllib/FixMyStreet/Cobrand/Camden.pm (hunk 1 above)
Comment (dracos) [reply perllib/FixMyStreet/Cobrand/Camden.pm:49]: Also there aren't any null entries, so not sure the whole function is needed :)

**Comment by dracos** [comment]
While you're here, as I've just done this for Brent, you'll want these two functions in the same file:
```
+sub open311_config {
+    my ($self, $row, $h, $params) = @_;
+    $params->{multi_photos} = 1;
+}
+
+sub open311_munge_update_params {
+    my ($self, $params, $comment, $body) = @_;
+    $params->{service_request_id_ext} = $comment->problem->id;
+}
```
First is because Symology open311-adapter accepts multiple photo URLs and puts them all in the description (think that's all it does), and the latter is because sending an update to Symology requires the FMS ID.

**Comment by chrismytton** [comment]
Thanks both! Have added an extra commit with the open311 hooks Matthew mentions.

**Pull request merged**